src/main/java/org/jala/efeeder/foodmeeting/ChangeFoodMeetingStatusCommand.java

```diff
@@ -35,18 +38,19 @@ public Out execute(In parameters) throws Exception {
 						.setIdFoodMeeting(receivedEvent.getIdFoodMeeting())
 						.setIdUser(receivedEvent.getIdUser())
 						.setNewStatus(receivedEvent.getNewStatus())
-						.setRedirectTo("/action/FoodMeeting")
 						.build()
 				)
 				.build()
 		);
-		MessageContext messageContext = MessageContext.newBuilder()
-				.setRoom(receivedMessage.getRoom())
+		
+		String homeRoomId = WebsocketsConstants.homeRoom;		
+		Builder messageBuilder = MessageContext.newBuilder()				
 				.setUser(receivedMessage.getUser())
-				.setEvents(events)
-				.build();
+				.setEvents(events);
+		
```

Comment:
I think this one is necessary to separate block of codes that are related to improve readability.